netket/hilbert/random/particle.py

```diff
@@ -32,7 +40,8 @@ def random_state(hilb: Particle, key, batches: int, *, dtype):
     boundary = np.tile(pbc, (batches, 1))
 
     Ls = np.array(hilb.n_particles * hilb.extent)
-    modulus = np.where(np.equal(pbc, False), jnp.inf, Ls)
+    modulus = np.where(np.equal(pbc, False), 1.0, Ls)
```

Comment:
can you clarify what is done in the not PBC case? (also add a more explicit description in the doc string)